keyvalue/server/src/main/java/alluxio/master/keyvalue/KeyValueMaster.java

```diff
@@ -441,7 +441,7 @@ private JournalEntry newMergeStoreEntry(long fromFileId, long toFileId) {
     return JournalEntry.newBuilder().setMergeStore(mergeStore).build();
   }
 
-  private Iterator<JournalEntry> getJournalEntryIterator(
+  private Iterator<JournalEntry> getStoreIterator(
```

Comment:
To avoid overloading function getJournalEntryIterator (the name is used by JournalEntryIterator interface).